src/quacc/recipes/espresso/dos.py

```diff
@@ -311,3 +358,88 @@ def projwfc_flow(
         "non_scf_job": non_scf_results,
         "projwfc_job": projwfc_results,
     }
+
+
+@flow
+def bands_flow(
+    atoms: Atoms,
+    job_decorators: dict[str, Callable | None] | None = None,
+    job_params: dict[str, Any] | None = None,
+) -> BandsSchema:
+    """
+    This function performs a bands calculation.
+
+    Consists of following jobs that can be modified:
+
+    1. pw.x static
+        - name: "static_job"
+        - job: [quacc.recipes.espresso.core.static_job][]
+
+    2. pw.x bands
+        - name: "bands_job"
+        - job: [quacc.recipes.espresso.core.bands_job][]
+
+    3. bands.x post processing
+        - name: "bands_pp_job"
+        - job: [quacc.recipes.espresso.dos.bands_pp_job][]
+
+    Parameters
+    ----------
+    atoms
+        Atoms object
+    job_params
+        Custom parameters to pass to each Job in the Flow. This is a dictinoary where
+        the keys are the names of the jobs and the values are dictionaries of parameters.
+    job_decorators
+        Custom decorators to apply to each Job in the Flow. This is a dictionary where
+        the keys are the names of the jobs and the values are decorators.
+
+    Returns
+    -------
+    BandsSchema
+        Dictionary of results from [quacc.schemas.ase.summarize_run][].
+        See the type-hint for the data structure.
+    """
+
+    static_job_defaults = {"kspacing": 0.2}
+    bands_job_defaults = recursive_dict_merge(
+        job_params.get("static_job", {}),
+        {"input_data": {"control": {"calculation": "bands", "verbosity": "high"}}},
```

Comment:
Maybe it makes sense to set `verbosity` to high in the `bands_job` function itself as a default?